### DISCUSSION TITLE
How to make a PUT/POST/GET/DELETE request?

### DISCUSSION THREAD

**Discussion by PickleCubeSoftwareLtd** [serverpod/serverpod#2266] [Q&A] [answered]
Hi,

I'm surprisingly struggling to find an answer on the web / documentation about this. 

We're looking at using serverpod for our backend solution to our mobile app. However, I need the capability of allowing GET/POST/PUT/DELETE operations on our server as some of our third party web hooks back to our API will require these. 

I understand serverpod's aim is to be a more RPC solution, but this seems like a big limitation if we cannot implement this. 

Does anyone have any pointers on if this is achievable or not? 

Thanks!

**Comment by Metaildust** [comment]
Have you tried http package from dart?

**Comment by binemmanuel** [comment]
Serverpod doesn't have any limitations to what you are trying to do as you can create methods in your endpoint to send HTTP requests with any METHOD (POST, GET, PUT, etc) using the http or dio package that you are already familiar with.

**Comment by vlidholt** [comment, marked as answer]
Hi! If you need to create very specific endpoints, you can use the Relic webserver that comes bundled with Serverpod.
https://docs.serverpod.dev/concepts/webserver

We are planning a large update to Relic, you can see the RFC here:
https://github.com/serverpod/serverpod/issues/2008
